addons/event_sale/views/event_views.xml

```diff
@@ -26,4 +26,71 @@
         </field>
     </record>
 
+    <record id="view_event_sale_report_graph" model="ir.ui.view">
+        <field name="name">event.sale.report.graph</field>
```

Comment:
Guidelines: name should be ``event.sale.report.view.graph`` (valid for other views)